src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -5460,15 +3336,15 @@ namespace AirflowNetworkSolver {
                     Htop = Z;
                     P = PZ + Dp;
                     if (Htop != Hbot) {
-                        Rho0 = PsyRhoAirFnPbTdbW(Pbz + P, T, X);
+                        Rho0 = AIRDENSITY(Pbz + P, T, X);
                         T += (Htop - Hbot) * BetaT;
                         X += (Htop - Hbot) * BetaXfct * X0;
-                        Rho1 = PsyRhoAirFnPbTdbW(Pbz + P, T, X);
+                        Rho1 = AIRDENSITY(Pbz + P, T, X);
                         BetaRho = (Rho1 - Rho0) / (Htop - Hbot);
                         Dp += psz(Pbz + P, Rho0, BetaRho, Hbot, Htop, G);
                     }
-                    RhoDr = PsyRhoAirFnPbTdbW(Pbz + PZ + Dp, T, X);
-                    Rho = PsyRhoAirFnPbTdbW(Pbz + PZ + Dp, T, X);
+                    RhoDr = AIRDENSITY(Pbz + PZ + Dp, T, X);
```

Comment:
We have an air density function in Psy module. Do we need an additional AIRDENSITY function?

Reply:
That's actually a `#define` to `PsyRhoAirFnPbTdbW`, so it is the same thing.